shared/types/src/dataset.ts

```diff
@@ -770,6 +770,7 @@ type SelectCohortEntry = {
 	values: SelectCohortValuesEntry[]
 	description?: string
 	/** subtext shown at the very bottom of the cohort/about tab subheader */
+	descriptionByUser?: { [index: string]: string }
```

Comment:
the implementation looks good. please fully document it even if it may refactored later. the `subtext shown at the very bottom of the cohort/about tab subheader` is for the `asterisk` property

Reply:
I am not sure what you mean with the subtext. I dont show anything at the bottom.